docs/csharp/programming-guide/statements-expressions-operators/expression-bodied-members.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Expression-bodied members (C# Programming Guide) | Microsoft Docs"
+ms.date: "2015-07-20"
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "expression-bodied members[C#]"
+  - "C# language, expresion-bodied members"
+author: "rpetrusha"
+ms.author: "ronpet"
+---
+# Expression-bodied members (C# programming guide)
+Expression body definitions let you provide a member's implementation in a very concise, readable form. You can use an expression body definition whenever the logic for any supported member, such as a method or property, consists of a single expression. An expression body definition has the following general syntax:
+
+```cs
+member => expression;
+```
+
+where *expression* is any valid expression. 
```

Comment:
I'd say 'a valid expression' instead of 'any valid expression'. I can't think of any disallowed off the top of my head, but I bet if I thought hard enough I could find one.